tests/unit/src/unit/TestRemoting.hx

```diff
@@ -12,9 +12,9 @@ class TestRemoting extends Test {
 	static var _ : Dynamic = init();
 	static var ecnx : haxe.remoting.ExternalConnection;
 	static var ecnx2 : haxe.remoting.ExternalConnection;
-	static var ecnx3 : haxe.remoting.ExternalConnection;
+	// static var ecnx3 : haxe.remoting.ExternalConnection;
```

Comment:
Yeah, though I'm kinda worried about that I could break something with this change, gotta review the diff again.